microsoft-edge/webview2/how-to/prerelease-testing.md

```diff
@@ -0,0 +1,246 @@
+
+Win32:
+* [ICoreWebView2ExperimentalEnvironmentOptions::get_ChannelSearchKind](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalenvironmentoptions#get_channelsearchkind)
+* [ICoreWebView2ExperimentalEnvironmentOptions::put_ChannelSearchKind](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalenvironmentoptions#put_channelsearchkind)
+* [COREWEBVIEW2_CHANNEL_SEARCH_KIND enum](/microsoft-edge/webview2/reference/win32/webview2experimental-idl#corewebview2_channel_search_kind)
+
+
+#### [Group policy](#tab/group-policy)
+
+Set the `ChannelSearchKind` policy.
+
+Name: `<app exe name or app user model ID - ex: WebView2APISample.exe>`  
+Value: `1`
+
+Do either of the following:
+* Download the Microsoft Edge policy files, which include the WebView2 policy files, from [Download and configure Microsoft Edge for Business](https://www.microsoft.com/edge/business/download).  Click any of the three **Download Windows Policy** links.<!-- todo: no "Edge policy" download link there -->
+* Use the built-in policy on Intune.<!-- todo: link to more info -->
+
+---
+
+
+<!-- ====================================================================== -->
+## Setting the preview channel by using the browser executable folder
+
+The browser executable folder is the path to the folder that contains the WebView2 Runtime binaries.  This option is useful if you're using a fixed Runtime.  The browser executable folder path can be set per app through a registry key, environment variable, API, or group policy.
+
+
+#### [Registry key](#tab/registry-key)
+
+```reg
+REG ADD <HKLM/HKCU>\Software\Policies\Microsoft\Edge\WebView2\BrowserExecutableFolder /v WebView2APISample.exe /t REG_SZ /d "C:\Users\myname\AppData\Local\Microsoft\Edge SxS\Application\88.0.680.0"
+```
+
+* Replace `WebView2APISample.exe` with your app's executable name, or with the application user model ID.
+
+* Replace the example path with the path to the desired Microsoft Edge preview channel.
+
+
+#### [Environment variable](#tab/environment-variable)
+
+Name: `WEBVIEW2_BROWSER_EXECUTABLE_FOLDER`  
+Value: `<path to desired Microsoft Edge preview channel>`
+
+
+#### [API](#tab/api)
+
+Pass the path to the browser executable folder during environment creation, by calling `CreateCoreWebView2EnvironmentWithOptions`.
+
+.NET: 
+* [todo]
+
+WinRT:
+* [todo]
+
+Win32:
+* [CreateCoreWebView2EnvironmentWithOptions](/microsoft-edge/webview2/reference/win32/webview2-idl#createcorewebview2environmentwithoptions)
+
+
+#### [Group policy](#tab/group-policy)
+
+Set `BrowserExecutableFolder`.
+
+Name: `<app exe name or app user model ID - ex: WebView2APISample.exe>`
+Value: `<path to desired Microsoft Edge preview channel>`
+
+Download the Microsoft Edge policy files, which include the WebView2 policy files, from the Edge Enterprise site,<!-- todo: link --> or use the built-in policy on Intune.<!-- todo: link -->
```

Comment:
copy Enterprise link from other tab.
copy Intune link from other comment in this PR.

Reply:
deleted that now-dup paragraph; use paragraph below it